packages/iTwinUI-react/.eslintignore

```diff
@@ -0,0 +1,13 @@
+**/*.json
+node_modules
+public
+styles
+.next
```

Comment:
so `json`, `public`, `styles`, `.next`, `coverage`, `dist` are all new and not sure if any of those are needed or even existing. What was the reason you have added `json`?

Reply:
i think it was trying to lint `turbo.json`. 

But I will need to look into this, as linting does not seem to work from command line

Reply:
I've cleaned up eslintignore so some of those (including `json`) have been removed